axelor-base/src/main/resources/views/RegistrationNumberTemplate.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<object-views xmlns="http://axelor.com/xml/ns/object-views"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://axelor.com/xml/ns/object-views http://axelor.com/xml/ns/object-views/object-views_6.1.xsd">
+
+  <grid name="registration-number-template-grid" title="Registration number templates"
+    model="com.axelor.apps.base.db.RegistrationNumberTemplate">
+    <field name="name"/>
+    <field name="validationMethodSelect"/>
+  </grid>
+
+  <form name="registration-number-template-form" title="Registration number templates"
+    model="com.axelor.apps.base.db.RegistrationNumberTemplate">
+    <panel name="mainPanel" title="Main Template">
+      <field name="name"/>
+      <field name="isRequiredForCompanies"/>
+      <field name="titleToDisplay"/>
+      <field name="requiredSize"/>
+      <field name="validationMethodSelect"/>
+
+      <panel title="Short registration number">
+        <field name="useSiren"/>
+        <field name="sirenPos" showIf="useSiren"/>
+        <field name="sirenLength" showIf="useSiren"/>
+      </panel>
+
+      <panel title="Internal classification number">
+        <field name="useNic"/>
+        <field name="nicPos" showIf="useNic"/>
+        <field name="nicLength" showIf="useNic"/>
+      </panel>
```

Comment:
The panels are not next to each other anymore because you added the field name
Fix the view please